Tests/scripts/slack_notifier.py

```diff
@@ -147,20 +147,19 @@ def get_attachments_for_bucket_upload_flow(build_url, job_name, packs_results_fi
         if successful_packs:
             steps_fields += [{
                 "title": "Successful Packs:",
-                "value": "\n".join([pack_name for pack_name in {*successful_packs}]),
+                "value": "\n".join(sorted([pack_name for pack_name in {*successful_packs}])),
                 "short": False
             }]
         if failed_packs:
             steps_fields += [{
                 "title": "Failed Packs:",
-                "value": "\n".join([f"{pack_name}: {PackStatus[pack_data.get(BucketUploadFlow.STATUS)].value}"
-                                    for pack_name, pack_data in failed_packs.items()]),
+                "value": "\n".join(sorted([pack_name for pack_name in {*failed_packs}])),
```

Comment:
Why we have decided to remove the status/reason here?

Reply:
It just looks bad and not so informative, I think it's better to have the slack msgs with the pack names only:
![image](https://user-images.githubusercontent.com/53565845/109418747-9c75f000-79d2-11eb-847b-713186eaf138.png)

Reply:
So why do we save it if we don't use it?